.github/workflows/pull_request.yaml

```diff
@@ -35,20 +54,25 @@ jobs:
         shell: bash
       - name: Set tox env for python 27
         if: matrix.python == '2.7'
-        run: echo "::set-env name=TOXENV::py27-static"
+        run: echo "::set-env name=TOXENV::py27-${{ matrix.env.TEST_TYPE }}"
       - name: Set tox env for python 36
         if: matrix.python == '3.6'
-        run: echo "::set-env name=TOXENV::py36-dynamic"
+        run: echo "::set-env name=TOXENV::py36-${{ matrix.env.TEST_TYPE }}"
       - name: Set tox env for python 37
         if: matrix.python == '3.7'
+        run: echo "::set-env name=TOXENV::py37-${{ matrix.env.TEST_TYPE }}"
+      - name: Pulling images for upgrade
+        if: matrix.env.TEST_TYPE == 'release-upgrade' || matrix.env.TEST_TYPE == 'source-upgrade'
         run: |
           docker pull quay.io/pulp/pulp-ci-c7:3.0.0
           docker pull quay.io/pulp/pulp-ci-dbuster:3.0.0
           docker pull quay.io/pulp/pulp-ci-f31:3.0.0
-          echo "::set-env name=TOXENV::py37-upgrade"
       - name: Run tox
         run: |
+          export TEST_TYPE
```

Comment:
Not sure it it is needed

Reply:
I'll try real quick; it'll fail quickly without it.